internal/impl/devbox.go

```diff
@@ -631,7 +631,7 @@ func (d *Devbox) RestartServices(ctx context.Context, serviceNames ...string) er
 
 	// TODO: Restart with no services should restart the _currently running_ services. This means we should get the list of running services from the process-compose, then restart them all.
 
-	svcSet, err := d.Services()
+	svcSet, err := d.Services("")
```

Comment:
The empty string can be hard to understand for a reader.

I'd suggest one of:
1. `d.Services("" /*userProcessCompose*/)`
2. define a separate function called `d.ServicesWithUserProcessCompose()`, and then `d.Services()` calls that with an empty-string.

Reply:
Option 3, make it an option for devbox.Open:

In internal/boxcli/services.go:

```golang
func startProcessManager(
	cmd *cobra.Command,
	args []string,
	servicesFlags servicesCmdFlags,
	flags serviceUpFlags,
) error {
	box, err := devbox.Open(&devopt.Opts{
		Dir:    servicesFlags.config.path,
                CustomProcessComposeFile: flags.processComposeFile,
		Writer: cmd.ErrOrStderr(),
	})
	if err != nil {
		return errors.WithStack(err)
	}

	return box.StartProcessManager(cmd.Context(), args, flags.background, flags.processComposeFile)
}
```

Another benefit of this method is we can eventually easily add this flag to other services commands (services ls, stop, etc)